eras/babbage/impl/CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## 1.6.0.0
 
+* Use `getAlonzoWitsVKeyNeeded` for `getWitsVKeyNeeded`
```

Comment:
Is it necessary to add this to the changelog? I thought we only have to add things that have changed at the type level.